src/domain/app/i18n/en.json

```
@@ -510,6 +510,7 @@
         "general": "If it is possible to register for the event, specify the start and end times of the registration here.",
         "volunteering": "If it is possible to register for the volunteering, specify the start and end times of the registration here."
       },
+      "infoTextEnvironmentalCertificate": "If the company or event has a certificate, its name is mandatory information (e.g. ecocompass). If there is no certificate, the certificate field does not indicate anything.",
```

```suggestion
      "infoTextEnvironmentalCertificate": "If the company or event has a certificate, its name is mandatory information (e.g. ecocompass). If there is no certificate, the certificate field is left blank.",
```
Felt that this would be closer to the meaning of the Finnish text.